### PR TITLE
20 shuffleboard widgets

### PR DESCRIPTION
**Pull request by Silvery-Sheen**
not 20 yet
the current commands were set to get the temperature values

### CODE REVIEW

File: src/main/java/frc/robot/Robot.java [hunk 1]
```diff
@@ -32,14 +42,23 @@ protected Robot() {
    */
   @Override
   public void robotInit() {
+    System.out.println("\t"+Utilities.deadzone(0, 0.5));
```
Comment (kcjesse02) [inline src/main/java/frc/robot/Robot.java:45]: What does this block of code you are adding do?

File: src/main/java/frc/robot/Robot.java [hunk 2]
```diff
@@ -67,28 +87,40 @@ public void robotPeriodic() {
...
    * This function is called periodically during test mode.
    */
   @Override
   public void testPeriodic() {
+    Scheduler.getInstance().run();
   }
+  
+  @Override
+  public void disabledInit() {
+    System.out.println("\t"+Utilities.deadzone(0, 0.5));
```
Comment (kcjesse02) [inline src/main/java/frc/robot/Robot.java:119]: Again, What does this do?

**Comment by kcjesse02** [comment]
good job

**Review by NickisNick123 — APPROVED** [review]
It is code and as long as it has been tested on the robot and working, it is good

**Pull request closed without merging**